src/cpp/session/modules/SessionProfiler.R

```diff
@@ -90,7 +90,7 @@
 {
    tryCatch({
       resources <- .rs.profileResources()
-      htmlFile <- normalizePath(tempfile(fileext = ".html", tmpdir = resources$tempPath), winslash = "/")
+      suppressWarnings(htmlFile <- normalizePath(tempfile(fileext = ".html", tmpdir = resources$tempPath), winslash = "/"))
```

Comment:
Thanks to both! I'm using `mustWork = FALSE` since it's more readable. Kevin, thanks for pointing this out; however, we currently only use `normalizePath` to fix the windows `\`, I've verified this operation is still performed even when the directory does not exists. Good to know there are cases where it might be needed for the file to exists.